mirgecom/gas_model.py

```diff
@@ -60,20 +71,12 @@
     TransportModel,
     GasTransportVars
 )
-from grudge.dof_desc import (
-    DD_VOLUME_ALL,
-    VolumeDomainTag,
-    DISCR_TAG_BASE,
-)
-import grudge.op as op
-from grudge.trace_pair import (
-    interior_trace_pairs,
-    tracepair_with_discr_tag
+from mirgecom.wall_model import (
+    WallEOS,
+    PorousFlowDependentVars
 )
 from mirgecom.utils import normalize_boundaries
 
-from typing import Optional
-
 
 @dataclass(frozen=True)
 class GasModel:
```

Comment:
Per our discussion, it's probably OK to leave this as is until we figure out what will actually be needed by the fluid operator in the end. I'll leave this unresolved though so Mike has a chance to see it.

Reply:
I agree with @majosm about the code construction druther, and also about that it is not a high-priority change at this point.   When we do get around to it, I think it'd be preferable to have a `MaterialModels` encapsulating structure that has a collection of domain-specific constructs, and/or a base `MaterialModel` that the domain-specific material models derive from.   It may not be a great fit to have a base class here... not sure.

Reply:
I also agree but I exactly wanted to avoid big changes. I plan on having an actual porous media flow implemented in mirge-com in the near future, possibly right after review. Can I create an issue to revisit this opportunely?

Reply:
I am _definitely_ ok with keeping things that work and refactoring them as time and opportunity allow.   The thing we are picking up on here though is the potential this strategy has for making future changes more difficult and ending up with a tangle of things we need to unravel later with considerably more difficulty.   I agree that we should avoid potentially messy or lengthy refactors when we are trying to make stuff work for prediction.  I like the idea of making an issue.

Reply:
Issue in #930